packages/api-client/src/types/variable.types.d.ts

```diff
@@ -0,0 +1,58 @@
+export interface CreateVariableRequest {
+  projectId: string
+  name: string
+  entries: Entries[]
+}
+export interface Entries {
+  value: string
+  environmentId: string
+}
+export interface CreateVariableResponse {
+  id: string
+  name: string
+  createdAt: string
+  updatedAt: string
+  note: string | null
+  lastUpdatedById: string
+  projectId: string
+  project: Project
+  versions: Entries[]
+}
+
+export interface Project {
+  workspaceId: string
+}
+export interface UpdateVariableRequest {
+  variableId?: string
+  name?: string
+  entries?: Entries[]
+}
+export interface UpdateVariableResponse {
+  variable: Variable
+  updatedVersions: Entries[]
+}
+
+export interface Variable {
+  id: string
+  name: string
+  note: string
+}
+export interface RollBackVariableRequest {
+  variableId: string
+  version: number
+  environmentId: string
+}
+export interface RollBackVariableResponse {}
```

Comment:
![image](https://github.com/user-attachments/assets/90e66ba1-ae36-4d35-bab4-24ee0dd54e46)

This is the response type/